adafruit_platformdetect/constants/chips.py

```diff
@@ -23,5 +23,5 @@
 MIPS24KEC = "MIPS24KEC"
 A64 = "A64"
 
-BCM_RANGE = {'BCM2708', 'BCM2709', 'BCM2835', 'BCM2837', 'bcm2708', 'bcm2709',
-             'bcm2835', 'bcm2837'}
+BCM_RANGE = {'BCM2708', 'BCM2709', 'BCM2711', 'BCM2835', 'BCM2837',
+             'bcm2708', 'bcm2709', 'bcm2711', 'bcm2835', 'bcm2837'}
```

Comment:
Hey @mruz, it is really needed to declared it in capital letters and lower letters? 

Why not doing something like the code below in `adafruit_platformdetect/chip.py`:
```python
if isinstance(hardware, str):
    if hardware.lower() in chips.BCM_RANGE:
        linux_id = chips.BCM2XXX
elif isinstance(hardware, list):
    if set([model.lower() for model in hardware]) & chips.BCM_RANGE:
        linux_id = chips.BCM2XXX
```

You can deside either to use the `lower()` or `upper()` string method so in the `BCM_RANGE` you just need capital letters or nor.

Reply:
Great! It should be working the same way now.